activiti-cloud-api-process-model-impl/src/main/java/org/activiti/cloud/api/process/model/impl/IntegrationRequestImpl.java

```diff
@@ -29,6 +29,11 @@ public IntegrationRequestImpl() {
 
     public IntegrationRequestImpl(IntegrationContext integrationContext) {
         this.integrationContext = integrationContext;
+        setAppVersionFromContext();
+    }
+
+    private void setAppVersionFromContext(){
+        this.setAppVersion(this.integrationContext.getAppVersion());
```

Comment:
a method for one line of code? so now we have 4 lines instead of a single one?

Reply:
inlined 👍